docs/hoon/concepts.md

```diff
@@ -179,31 +180,32 @@ A `mold` is a constructor function (`gate`).  Its sample is any
 noun; its product is a structured noun.  A mold is idempotent;
 `(mold (mold x))` always equals `(mold x)`.
 
-Since there's no such thing as a [`span`](#span) declaration, we use molds
-to define useful spans by example; we also use them to validate
-untrusted network input.
+Because there's no such thing as a [`type`](#type) declaration, 
+we use molds to define useful types by example; we also use them 
+to validate untrusted network input.
 
 ### `face` (named variable)
 
 In a conventional language, we have a scope, environment or
 symbol table.  Declaring a variable, like `var foo: atom`, adds
 the name `foo` to the table with type `atom`.
 
-The Hoon equivalent is `:var  foo  atom`.   But Hoon has a
+The Hoon analogue is `=|(foo atom)`.   But Hoon has a
```

Comment:
This would be something like `=/  foo=atom` or `=/(foo=atom 1 ...)`. `=|(foo atom)` is "pin the bunt of *foo* and produce *atom*". But I'm not sure if the example is clarifying here ...

Reply:
Good point. Another possibility would be `=|(foo=atom foo)`, which will print `0`.

Reply:
I was just trying to match the pseudo-code `var foo: atom`, which defines foo without giving it a value.  We don't exactly have that, so I tried to give the closest approximation.  I'm open to ideas about how to make that section clearer and more helpful.

Reply:
`=|` is indeed hoon's way to declare a variable with minimal initialization, so it is the closest approximation. I think a more generally useful concept to translate from other languages is `var foo: atom = 3`, though, which would correspond to `=/` or `=+`, and of those two `=/` is probably the better choice since it lets you express the type of the variable in tall form. Either way is fine, but it should be correct syntax, which requires having the third argument to `=|`. Maybe something like: `=|(foo=atom <sub-hoon>)`

Reply:
Change made